### PR TITLE
Fix status reports

### PR DESCRIPTION
**Pull request by timgl**
## Changes

The instance status report was broken in 1.28.1.

I don't think these tests ever properly ran. Moved them into the right configuration and fixed the issues.

## How did you test this code?

*Please describe.*

### CODE REVIEW

File: ee/tasks/test/test_status_report.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    )
+
+
+def _create_person(distinct_id: str, team: Team) -> Person:
+    person = Person.objects.create(team=team, distinct_ids=[distinct_id])
+    return Person(id=person.uuid)
+
+
+class TestStatusReport(factory_status_report(_create_event, _create_person)):  # type: ignore
+    # CH only
```
Comment (yakkomajuri) [inline ee/tasks/test/test_status_report.py:33]: comment not needed anymore

File: ee/tasks/test/test_status_report.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    def test_status_report_duplicate_distinct_ids(self) -> None:
+        create_person_distinct_id(self.team.id, "duplicate_id1", str(UUIDT()))
+        create_person_distinct_id(self.team.id, "duplicate_id1", str(UUIDT()))
+        create_person_distinct_id(self.team.id, "duplicate_id2", str(UUIDT()))
+        create_person_distinct_id(self.team.id, "duplicate_id2", str(UUIDT()))
+        create_person_distinct_id(self.team.id, "duplicate_id2", str(UUIDT()))
+
+        for index in range(0, 2):
+            sync_execute(
+                "INSERT INTO person_distinct_id SELECT %(distinct_id)s, %(person_id)s, %(team_id)s, 1, %(timestamp)s, 0 VALUES",
```
Comment (yakkomajuri) [inline ee/tasks/test/test_status_report.py:43]: nit: I sort of get why but it's a bit weird that we create distinct ids in different ways

**Review by yakkomajuri — APPROVED** [review]
overall looks good to me, thanks for picking this up

wondering if we should handle individual queries failing while allowing the report to proceed, but that could go in another PR.

(while this was broken, queries could fail because of e.g. a timeout, and then we don't send the report)

File: ee/tasks/test/test_status_report.py (hunk 2 above)
Comment (timgl) [reply ee/tasks/test/test_status_report.py:43]: yeah the other method didn't allow me to set timestamps and I didn't want to edit the production code just for the test

File: ee/tasks/test/test_status_report.py (hunk 1 above)
Comment (timgl) [reply ee/tasks/test/test_status_report.py:33]: agree but i don't want to wait another 11 mins for the tests to finish 😂

**Pull request merged**